database/raft/membership.go

```diff
@@ -0,0 +1,22 @@
+package raft
+
+import "context"
+
+const potentialMemberPrefix = "potential"
```

Comment:
1. Ah, yes, I'll add in `/`. 
2. What about `allowed`? 
3. I think that's a good idea--I forgot about it, basically.